library/std/src/os/windows/fs.rs

```diff
@@ -472,10 +472,21 @@ pub trait MetadataExt {
     #[unstable(feature = "windows_by_handle", issue = "63010")]
     fn file_index(&self) -> Option<u64>;
 
-    /// Returns the change time, which is the last time file metadata was changed, such as
-    /// renames, attributes, etc
-    ///
-    /// This will return `None` if the `Metadata` instance was not created using the `FILE_BASIC_INFO` type.
+    /// Returns the value of the `ChangeTime{Low,High}` field from the
+    /// [`FILE_BASIC_INFO`] struct associated with the current file handle.
+    /// [`ChangeTime`] is the last time file metadata was changed, such as
+    /// renames, attributes, etc.
+    ///
+    /// This will return `None` if `Metadata` was populated without access to
+    /// [`FILE_BASIC_INFO`]. For example, the result of `std::fs::read_dir`
+    /// will be derived from [`WIN32_FIND_DATA`] which does not have access to
+    /// `ChangeTime`.
+    ///
+    /// [`FILE_BASIC_INFO`]: https://learn.microsoft.com/windows/win32/api/winbase/ns-winbase-file_basic_info
+    /// [`WIN32_FIND_DATA`]: https://learn.microsoft.com/windows/win32/api/minwinbase/ns-minwinbase-win32_find_dataw
+    /// [`FindFirstFile`]: https://learn.microsoft.com/windows/win32/api/fileapi/nf-fileapi-findfirstfilea
+    /// [`FindNextFile`]: https://learn.microsoft.com/windows/win32/api/fileapi/nf-fileapi-findnextfilea
+    /// [`ChangeTime`]: https://devblogs.microsoft.com/oldnewthing/20100709-00/?p=13463.
```

Comment:
Raymond's blog is not authoritative and ships in only English, so am not a fan of linking it here. Also not a fan of the linkage to ANSI APIs, which increases their exposure. Are these docs needed? Feels like they're leaking internal implementation details.

Reply:
So I got my PR merged, but it's still not showing up on the official site yet, https://github.com/MicrosoftDocs/sdk-api/pull/1863

Reply:
>Also not a fan of the linkage to ANSI APIs, which increases their exposure. Are these docs needed? Feels like they're leaking internal implementation details.

I added that to explain in what cases this extension can return None from feedback that this was unclear from the original docs. Do you have a recommendation?

Reply:
I would be happy to retain these links for internal reference (i.e. as comments not doc comments).

Reply:
Ok I'll fix it so that it looks more like the extension methods above it and remove the links.